utils/src/requests/QueryAllSubjectDTO.ts

```diff
@@ -49,4 +49,4 @@ export class QueryAllSubjectDTO {
   })
   @IsOptional()
     groupId?: string;
-}
\ No newline at end of file
+}
```

Comment:
This dto must inherit properties from QueryAllDTO, then some of the properties here can be deleted (page, pageSize, search, order). After that you need to update the messages in documentation.